includes/admin/load.php

```diff
@@ -32,6 +34,8 @@ function perflab_add_features_page() {
 
 	return $hook_suffix;
 }
+
+// @phpstan-ignore-next-line
```

Comment:
Because PHPStan identifies correctly that this function is used as an action, and not a filter. So it shouldn't return anything.

Reply:
The returned value is used in testing.

Reply:
Ideally the tests could be updated to not rely on this, but I think that is out of scope for this PR.

Reply:
Follow-up: https://github.com/WordPress/performance/pull/1198/files#r1589800613